doc/source/configuration/security-hardening.rst

```diff
@@ -0,0 +1,44 @@
+==================
+Security Hardening
+==================
+
+CIS Benchmark Hardening
+-----------------------
+
+The roles from the `Ansible-Lockdown <https://github.com/ansible-lockdown>`_
+project are used to harden hosts in accordance with the CIS benchmark criteria.
+It won't get your benchmark score to 100%, but should provide a significant
+improvement over an unhardened system. A typical score would be 70%.
```

Comment:
I suppose we should mention how to find out your score (Wazuh). Could be a follow up docs change (which would use CI-lite)

Reply:
> I suppose we should mention how to find out your score (Wazuh). Could be a follow up docs change (which would use CI-lite)

Bumping, in case this one got lost.